src/examples/ImageExamplePage.tsx

```diff
@@ -0,0 +1,90 @@
+'use strict';
+import {Image} from 'react-native';
+import React from 'react';
+import {Example} from '../components/Example';
+import {Page} from '../components/Page';
+import {LinkContainer} from '../components/LinkContainer';
+
+export const ImageExamplePage: React.FunctionComponent<{}> = () => {
+  const example1jsx = `import {Image} from 'react-native';
+  
+<Image
+  style={{width: 50, height: 50}}
+  source={{
+    uri: 'https://reactnative.dev/img/tiny_logo.png',
+  }}/>`;
+  const example2jsx = `import {Image} from 'react-native';
+  
+<Image
+  style={{width: 75, height: 50, resizeMode: 'stretch'}}
+  source={{
+    uri: 'https://reactnative.dev/img/tiny_logo.png',
+  }}/>`;
+  const example3jsx = `import {Image} from 'react-native';
+  
+<Image
+  style={{width: 300, height: 50, resizeMode: 'repeat'}}
+  source={require('../assets/tiny_logo.png')}
+/>`;
+  const example4jsx = `import {Image} from 'react-native';
+  
+<Image
+  style={{width: 75, height: 50, resizeMode: 'cover'}}
+  source={require('../assets/tiny_logo.png')}/>`;
+
+  return (
+    <Page
+      title="Image"
+      description="A component which scales and displays images.">
+      <Example title="A simple Image from web source." code={example1jsx}>
+        <Image
+          style={{width: 50, height: 50}}
+          source={{
+            uri: 'https://reactnative.dev/img/tiny_logo.png',
+          }}
+        />
+      </Example>
+      <Example
+        title="An Image from web source stretched to fill view."
+        code={example2jsx}>
+        <Image
+          style={{width: 75, height: 50, resizeMode: 'stretch'}}
+          source={{
+            uri: 'https://reactnative.dev/img/tiny_logo.png',
```

Comment:
>https://reactnative.dev/img/tiny_logo.png [](start = 18, length = 41)

I suggest using an image from this repo, as internet URIs have a way of not staying stable over time.

There is a trick for this, which I used to create this URI for you:  [https://user-images.githubusercontent.com/33470154/104789118-fa16eb80-5748-11eb-9870-68360eca6fa6.png](https://user-images.githubusercontent.com/33470154/104789118-fa16eb80-5748-11eb-9870-68360eca6fa6.png)
-Create a new issue on the repo
-Drag n drop an image into the description of the issue
-Submit
Now you have a persistent image that isn't tied to a branch.

Reply:
Great sounds good I'll make that change now!